Dockerfile

```diff
@@ -1,44 +1,36 @@
-# syntax=docker/dockerfile:1.2
-# Stream swaps to CentOS Stream once and is reused.
-# FROM quay.io/centos/centos:stream8 AS stream
-FROM registry.access.redhat.com/ubi8/ubi:latest as base
-RUN set -ex\
-	; dnf install -y gpgme-devel \
-	; dnf -y -q clean all
-
-RUN dnf install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False git perl wget make gcc
-
-
-FROM base AS build
-
-ARG BUILDER_SRC
-
-ENV GO_VERSION=1.16.13
-ENV GO_OS=linux
-ENV GO_ARCH=amd64
-ENV GOPATH=/go
-
-RUN curl https://dl.google.com/go/go${GO_VERSION}.${GO_OS}-${GO_ARCH}.tar.gz --output go.tar.gz
-RUN tar -C /usr/local -xzf go.tar.gz > /dev/null
-ENV PATH=$PATH:/usr/local/go/bin:${GOPATH}/bin
+FROM golang:1.15-alpine3.12 as build
+RUN apk --no-cache add build-base git
+COPY . /go/src/
+RUN cd /go/src/ && env GOOS=linux GOARCH=amd64 make build
 
-RUN go version
 
-COPY . /go/src/${BUILDER_SRC}
-RUN cd /go/src/${BUILDER_SRC} && go mod vendor && make build
-
-
-FROM base AS final
+FROM quay.io/podman/stable:v3.4.4
```

Comment:
We don't really need to use the podman image right?
This is just for running the go binary. Can we use something like ubi8-minimal and install the bare minimum: git.
The reason I'm trying to minimize the image size is that each builders need to do a full pull of this image, every time.

Reply:
I've since updated to use Redhat registry images, the new size is 536MB